lib/components/schematic-text.ts

```diff
@@ -5,5 +5,7 @@ export const schematicTextProps = z.object({
   schX: distance,
   schY: distance,
   text: z.string(),
+  anchor: z.enum(["center", "left", "right", "top", "bottom"]).optional(),
```

Comment:
You should actually use a “nine point anchor” here, we should have an example… one sec

Reply:
It looks like we do it wrong even in circuit json, although circuit json is more correct.

Here’s a full nice point anchor

```tsx

type AnchorAlignment =
    | "top_left"
    | "top_center"
    | "top_right"
    | "center_left"
    | "center"
    | "center_right"
    | "bottom_left"
    | "bottom_center"
    | "bottom_right";

```

![image](https://github.com/user-attachments/assets/ac4ee458-2575-4cbb-abb4-6995a2b3aefd)

Reply:
I would also support left_center and right_center as aliases (with a transform). We can put nine_point_anchor into lib/common